mars/tensor/datasource/execute_util.py

```diff
@@ -0,0 +1,31 @@
+#!/usr/bin/env python
```

Comment:
This file is not required any more. `execute` method has been added to `tensor.datasource.array.ArrayDataSource`.

Reply:
Rename `_create_array`  to `create_array` and put it into `array_utils.py`, no need to create a new file for a single util function.